web-client/src/components/information.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import '../App.css';
+
+function Information(props) {
+    const styles = makeStyles({
+        main:
+        {
+            padding: '5%',
+            width: '60%',
+        },
+        left:
+        {
+            width: '30%',
+            float: 'left',
+        },
+        right:
+        {
+            float: 'left',
+            marginTop: '2%',
+        },
+        photo:
+        {
+            display: 'block',
+            width: '60%',
+            borderRadius: '50%',
+        },
+        text:
+        {
+            fontWeight: 'normal',
+        },
+        a:
+        {
+            textDecoration: 'none',
+            color: 'black',
+
+            '&:hover' :
+            {
+                color: 'red',
+            }
+        }
+    });
+
+    const style = styles();
+
+    return (
+        <div className={style.main}>
+            <div className={style.left}>
+                <img src={props.path} className={style.photo} alt="not found!"></img>
```

Comment:
alt describes what is in the image so we should provide this info (because if someone uses screen reader, this info won't help him)

Reply:
I changed the alt values

Reply:
ok but I guess later we won't know that "icon" is for alt text (maybe other name?)

Reply:
ok no problem